test/Concurrency/async_task_locals_basic_warnings_bug_isolation.swift

```diff
@@ -13,9 +13,6 @@ actor Test {
   func withTaskLocal(isolation: isolated (any Actor)? = #isolation,
                      _ body: (consuming NonSendableValue, isolated (any Actor)?) -> Void) async {
     Self.$local.withValue(12) {
-      // Unexpected errors here:
```

Comment:
NOTE: We already did not emit this error. But this PR exercised the code in this area and when I was working on it the error came back, so I looked at it. The comment here is from when we still emitted the error here (and I put in my previous fix), so I removed it to prevent any confusion.